client/src/components/Searchbar/Searchbar.spec.js

```diff
@@ -0,0 +1,60 @@
+import { render, screen, fireEvent } from "@testing-library/react";
+import Searchbar from "./Searchbar";
+
+describe("Searchbar component", () => {
+  it("shows a dropdown to choose field of competence and input field for entering searchterms", () => {
+    const searchterm = {
+      competence: "",
+      buzzwords: "",
+    };
+    render(<Searchbar searchterm={searchterm} />);
```

Comment:
We got an error when running this test. 
Your Searchbar form is a controlled component and needs the handleChange function.
You need to add a dummy function so that the testing library is happy
```suggestion
    const searchterm = {
      competence: "",
      buzzwords: "",
    };
    
    const handleChange = jest.fn(() => {} )
    render(<Searchbar searchterm={searchterm} onHandleChange={handleChange} />);
```